src/commonMain/kotlin/io/modelcontextprotocol/kotlin/sdk/server/Server.kt

```diff
@@ -111,8 +125,9 @@ public open class Server(
         get() = _resources.value
 
     init {
-        logger.debug { "Initializing MCP server with capabilities: $capabilities" }
+        logger.debug { "Initializing MCP server with option: $options" }
 
+        // TODO: Remove all after Protocol inheritance
```

Comment:
please log an issue for TODO and mention it in the comment (same for other TODOs)